gpu-operator/platform-support.rst

```diff
@@ -165,6 +165,16 @@ The following NVIDIA data center GPUs are supported on x86 based platforms:
     | NVIDIA T400             | Turing                 |
     +-------------------------+------------------------+
 
+  .. tab-item:: G-series Products
+
+    +-------------------------+------------------------+
+    | Product                 | Architecture           |
+    +=========================+========================+
+    | NVIDIA HGX B200         | NVIDIA Blackwell       |
```

Comment:
Is HGX B200 a G-series product? Actually, I am not even sure what G-series means here

Reply:
let me check on this

Reply:
Thanks!